### DISCUSSION TITLE
How to get the contents from the editor

### DISCUSSION THREAD

**Discussion by eokwukwe** [vueup/vue-quill#438] [Q&A]
I am new to Quill Editor. I have been able to add it to my project using this vue-quill package. But I need help seeing how to get the contents typed in the editor so that I can save them for display later on another page. Your help will be appreciated.

**Comment by glaures** [comment]
Same problem here. I can see the event with some arrays and deltas in it. Is there a way to get the entire content as a whole?

**Comment by andradedaniel** [comment]
That issue help me: 
https://github.com/vueup/vue-quill/issues/33

**Comment by andradedaniel** [comment]
Complete example with Composition API: 

```
<script setup>
import { QuillEditor } from '@vueup/vue-quill'
import '@vueup/vue-quill/dist/vue-quill.snow.css'
import { ref } from 'vue'

const myEditor = ref(null)

function handleTextChange() {
    console.log(myEditor.value.getHTML())
}
</script>

<template>
    <QuillEditor
        ref="myEditor"
        @text-change="handleTextChange"
        theme="snow"
        toolbar="essential"
        content-type="html"
    />
</template>
```

The `text-change` event will be triggered when user type something or any toolbar button clicked

**Reply by ibnusyawall** [reply]
Wow, thanks dude!, that's working perfectly for me.
